### ISSUE TITLE
Republish validation errors from debug pings to dedicated topic

### ISSUE THREAD

**Issue by akkomar** [mozilla/gcp-ingestion#528]
With #517 we have a general mechanism to separate "debug" pings.
It would be useful for consumers of this stream to have a corresponding, filtered error topic containing only errors from pings tagged with debug header.

**Comment by akkomar** [comment]
cc @jklukas @whd  - opinions?

**Comment by jklukas** [comment]
This could be handled without any additional code changes by deploying an additional Republisher instance for the structured decoder error topic. It's additional operational overhead, though.

If we implemented https://github.com/mozilla/gcp-ingestion/issues/423, I expect that would handle this case. We could have a field on that table for x_debug_id and you could query BQ with a WHERE clause for that field being non-null.

Can you explain a bit more about your expected use? Would you be actually streaming these error into the debug view app and visualizing there?

**Comment by akkomar** [comment]
Yes, this is related to DebugView (see https://github.com/mozilla/debug-ping-view/issues/5, currently involving pulling whole `decoded-error` topic).

#423 wouldn't help me because of the latency. I would treat the topic proposed here as an analog of `decoded-debug` one - I think applications consuming `decoded-debug` stream would usually also benefit from its `-error` counterpart.
> (...) It's additional operational overhead (...)

We can hold on with that until we have concrete plans for more real-time-debug applications (this is likely in the coming months).

**Comment by jklukas** [comment]
> I think applications consuming decoded-debug stream would usually also benefit from its -error counterpart

In that case, maybe we want to republish debug errors into the `decoded-debug` topic and at least avoid the overhead of managing another topic and set of subscriptions. It would then be the consuming application's responsibility to partition out errors from valid messages based on attributes.

**Comment by akkomar** [comment]
I wouldn't mix different classes of messages in a single topic. At least not yet, until we have some more consumers for both of them - maybe after all it will turn out that debug messages are exceptional and it makes sense here.
